### ISSUE TITLE
Running interface.py "No checkpoint file."

### ISSUE THREAD

**Issue by aliericcantona** [hellochick/ICNet-tensorflow#47]
I downloaded from the google drive link ade20K includes 5 files one of them is checkpoint.txt
I dropped them on ./model

Then:
python inference.py --img-path=./input/outdoor_1.png --dataset=ade20k --filter-scale=2 --model=others

It says:
030] Creating TensorFlow device (/gpu:1) -> (device: 1, name: GeForce GTX 980, pci bus id: 0000:82:00.0)
No checkpoint file found.

do we have to change anything on interface.py script?

**Comment by aliericcantona** [comment]
Another attempt, I installed tensorflow-gpu 1.6.0 for CUDA library 8.0 but still running python inference.py --img-path=./input/outdoor_1.png --dataset=ade20k --filter-scale=2 --model=others 

generates "no checkpoint file found" and the results are not he same as the one stated here.

**Comment by hellochick** [comment]
Hey @aliericcantona, it seems that the checkpoint is missing, so you cannot load the pre-trained weights. You can refer to #45.

**Comment by srcolinas** [comment]
change checkpoints.txt to checkpoints

**Comment by aliericcantona** [comment]
thanks it works. Actually checkpoint.txt --> checkpoint

**Comment by aliericcantona** [comment]
the results are not exactly the same as the one you posted just slightly different (may be because of the slightly trained model difference). The color model (labels) are exactly the same as PSPNET ADE20K or different?

**Comment by hellochick** [comment]
@aliericcantona, I have updated the model but the output results not, so the results would be a little different.

**Comment by wangchuanya** [comment]
# If you want to apply to other datasets, change following four lines

IGNORE_LABEL = 255  # The class number of background
INPUT_SIZE = '720, 720'  # Input size for training

BATCH_SIZE = 16
LEARNING_RATE = 1e-3
MOMENTUM = 0.9
NUM_CLASSES = 19
NUM_STEPS = 60001
POWER = 0.9
RANDOM_SEED = 1234
WEIGHT_DECAY = 0.0001
PRETRAINED_MODEL = 
SNAPSHOT_DIR = './snapshots/'
SAVE_NUM_IMAGES = 4
SAVE_PRED_EVERY = 50

these codes in train.py，in model file has ade20k weights,but failed to train, PRETRAINED_MODEL  how to load ,PRETRAINED_MODEL = ?
